service.subtitles.titlovi-com-official/addon.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.subtitles.titlovi-com-official"
+       name="Titlovi.com Official "
+       version="1.0.0"
+       provider-name="titlovi.com">
+    <requires>
+        <import addon="xbmc.python" version="2.7"/>
+        <import addon="script.module.requests" version="2.18.4"/>
+        <import addon="script.module.simplecache"/> 
+    </requires>
+    <extension point="xbmc.subtitle.module" library="service.py"/>
+    <extension point="xbmc.addon.metadata">
+        <platform>all</platform>
+        <summary lang="en">Titlovi.com</summary>
+        <description lang="en">Search and download Titlovi.com subtitles. Click here to read more info about manual searching.</description>
+        <disclaimer>Users need to provide Titlovi.com username and password in add-on configuration</disclaimer>
+        <news>
+Official addon
+-When using manual search for TV shows please use this example format for season and episode numbers: The Wire S01E01
+        </news>
+        <platform>all</platform>
+        <language></language>
+        <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
+        <website>http://www.titlovi.com</website>
+        <email>tturka33 [at] gmail {dot} com</email>
+        <assets>
+            <icon>icon.png</icon>
+            <fanart>fanart.jpg</fanart>
+			<screenshot>resources/media/screenshot1.png</screenshot>
+            <screenshot>resources/media/screenshot2.png</screenshot>
+            <screenshot>resources/media/screenshot3.png</screenshot>
```

Comment:
those filenames are case sensitive (your screenshot have a capital extension)

Reply:
converting them to a jpg format is also appreciated as they are smaller

Reply:
Thanks for the review, i figured that extensions should be lowercase...